kroma-validator/cmd/README.md

```diff
@@ -6,86 +6,110 @@ the new validator system with the asset token are included.
 
 ## Usage
 
-You can use these subcommands via running kroma-validator container.
+You can use these subcommands via the running kroma-validator container.
 
 ```bash
-docker compose exec kroma-validator validator [command]
+docker compose exec kroma-validator kroma-validator [command]
```

Comment:
This was intended because the names are same (you can refer [this](https://github.com/kroma-network/kroma/pull/302#issuecomment-2081916790))

Reply:
Can you check [this comment](https://github.com/kroma-network/kroma/pull/317#discussion_r1592364832)?